Makefile

```
@@ -21,8 +21,7 @@ test:
 		-testPlan $(SCHEME) \
 		-skipPackagePluginValidation \
 		CODE_SIGNING_REQUIRED=NO \
-		CODE_SIGNING_ALLOWED=NO \
-		| LC_ALL="en_US.UTF-8" xcpretty
```

I was debugging a CI-only test failure and `xcpretty` was swallowing a helpful error log - `xcbeautify` seems like it provides better output in general for tests.

For context, the default xcodebuild output is thousands of lines, and not at all readable.